P5/Source/Guidelines/en/FS-FeatureStructures.xml

```diff
@@ -352,8 +367,22 @@ each one represented as an identifiable <gi>fs</gi> element within a
 Other <gi>f</gi> elements may invoke them by reference, using the
 <att>fVal</att> attribute; for example, one might use them in a
 feature value pair such as: <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#UND"><f name="dental-fricative" fVal="#T.DF"/> </egXML> rather than expanding the hierarchy of the
-component phonological features explicitly.  </p>
-<p>Feature structures stored in this way  may also be associated with
+component phonological features explicitly.</p>
+<p>The feature structure that concludes section <ptr target="#FSSY"/> above, identifying the
+value of some part of speech to be a "common noun" can be used in tandem with a
```

Comment:
I’d recommend using an element instead of the straight quotes. Maybe `<soCalled>common noun</soCalled>` (or  `<q>`)

Reply:
I agree completely, and the same goes for lines 2664 & 2665 in DI chapter (previous file), too. I do not think `<soCalled>` is appropriate, though. My instinct is either `<q>` or `<term>`?
In any case, though, the comma afterwards is missing (before “can” which perhaps should be “may”), and the sentence itself feels a bit run-onny:

> The feature structure that concludes section [FSSY] above, identifying the value of some part of speech to be a “common noun” can be used in tandem with a feature-value library, which offers a way to encode a grammatical “tagset”, in this case containing labels for parts of speech: [EXAMPLE]

Reply:
sigh, true, I wonder how many straight quotes I have left there... that may take a while ;-)

Reply:
Tried to use `<q>` for quoted bits, and `<soCalled>` for, e.g., names of concepts. Wow, semantic markup... cool. :-)

Reply:
And it's just struck me that maybe (just maybe) some of those could be `<term>`s, but I think that sorting that out needs a bit of quiet and focus, and some target framework, such as that sketched in #2358 -- where you'd, for example, get a pop-up over a `<term>`, citing its definition.

Reply:
I believe I have eliminated simple quotes in the spec and in the FS chapter, with `<q>` where the quotes were straightforward. I even ended up removing quotes where they were not really needed, it seems. 
There are two sentences in FS that feel "run-onny" (if I interpret that correctly) -- in both cases they talk about some functionality that should/may accompany a suggestion exemplified in an earlier section. The potentially unstable word in both is "concludes", because one could imagine the target section to grow, one day. But today, they are true, and I think that when the FS chapter receives a solid revision, these two instances will be modified if needed.

Reply:
... and I've touched the DI chapter as well.